tool_path_planner/include/tool_path_planner/plane_slicer_raster_generator.h

```diff
@@ -65,6 +70,12 @@ class PlaneSlicerRasterGenerator : public PathGenerator
       jv["min_segment_size"] = min_segment_size;
       jv["search_radius"] = search_radius;
       jv["min_hole_size"] = min_hole_size;
+      jv["raster_wrt_global_axes"] = raster_wrt_global_axes;
+      Json::Value raster_dir(Json::ValueType::objectValue);
+      raster_dir["x"] = raster_direction.x();
+      raster_dir["y"] = raster_direction.y();
+      raster_dir["z"] = raster_direction.z();
+      jv["raster_direction"] = raster_dir;
```

Comment:
I think this will work, yaml might've been a better choice as it can directly casts to std vectors and maps